src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/coding/fallthrough/InputFallThrough.java

```diff
@@ -43,7 +43,7 @@ void tryResource() throws Exception {
                 return;
             }
             finally {
-            } // ok
```

Comment:
This is a good one to keep, please do something like `// ok, either exception is thrown or we return from method before fallthrough can occur`

Reply:
done